src/Modal.js

```diff
@@ -68,7 +68,7 @@ class Modal extends React.Component {
     keyboard:           true,
     animate:            true,
     transition:         true,
-    container:          document.body,
+    container:          typeof window != 'undefined' ? document.body : null,
```

Comment:
there is actually a util for doing this already on the page you can do: `canUseDOM ?  document.body : null`